### ISSUE TITLE
[ Help ] Android = How create landscape splash image that does not get stretched ?

### ISSUE THREAD

**Issue by esutton** [crazycodeboy/react-native-splash-screen#168]
Thank you for a great lib!

I took your two Android example app images and edited.  Very easy!

Thank you for sharing your work!

**Comment by weisjohn** [comment]
@esutton did you figure this out?

**Comment by weisjohn** [comment]
@esutton check this out: https://github.com/crazycodeboy/react-native-splash-screen/pull/141

**Comment by esutton** [comment]
@weisjohn No. I tried opening my react native project in Android Studio and try to understand layouts.

I copied  \layout\launch_screen.xml and was experimenting with layouts using buttons and such as place for images. 

I wanted one image to stay centered vertically and horizontally.
And then a green title bar at the top with another image centered vertically and horizontally. 

Anyway, I do not understand Android Studio layouts.   I am going to have to find some tutorials to watch.

**Comment by kunalqss** [comment]
Were you able to fix this ?

**Comment by esutton** [comment]
@kunalqss I have not figured it out.

**Comment by kunalqss** [comment]
In my case, the image is stretched for about a second and then gets back to its original size (in landscape mode). I have removed the explicit width and height given to the image but still unable resolve this issue.

**Comment by thedanielforum** [comment]
I am having the same issue as @kunalqss, do we know of any fix for this yet?

**Comment by hpthang88** [comment]
I have same issue

I'm fixed it by:
- Create new folder: android/src/res/layout-land
- Add file launch_screen.xml and content file as file /laypout/launch_screen.xml, then change android:background="@drawable/launch_screen_land"
- Create images `launch_screen_land.png` in `drawable-xxx` folders. Image's size is fit landscape screen size

**Comment by terrysahaidak** [comment]
Here how I've managed to show simple small image with logo on a white screen:

```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="fill_parent"
    android:layout_height="fill_parent"
    android:background="@color/white"
    android:orientation="vertical" >

    <LinearLayout
        android:id="@+id/lin_all"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerHorizontal="true"
        android:layout_centerVertical="true"
        android:layout_marginLeft="105dp"
        android:layout_marginRight="105dp"
        android:orientation="vertical" >
            <ImageView
                android:src="@drawable/launch_screen"
                android:layout_width="wrap_content"
         android:layout_height="wrap_content"
                android:scaleType="fitCenter"
            >
            </ImageView>
    </LinearLayout>
</RelativeLayout>
```

**Comment by darriuk** [comment]
Thank you *so much* Terry this worked really well for me.
